apps/gateway/src/common/utils/setup-app.ts

```diff
@@ -0,0 +1,18 @@
+import { GrpcExceptionFilter } from '../filters/grpc-exception.filter';
+import { HttpExceptionFilter } from '../filters/http-exception.filter';
+import { INestApplication, ValidationPipe } from '@nestjs/common';
+
+export default function setupApp(app: INestApplication): void {
+  app.enableCors();
+
+  app.useGlobalFilters(new GrpcExceptionFilter(), new HttpExceptionFilter());
+
+  const globalPrefix = 'api';
+  app.setGlobalPrefix(globalPrefix);
```

Comment:
you do not reuse the variable, and method name explaines what it needs so just
```suggestion
  app.setGlobalPrefix('api');
```